packages/rebox/ios/src/build-ios-app-debug.ts

```diff
@@ -31,7 +31,6 @@ export const buildIosAppDebug = async (options: TBuildIosAppDebugOptions): Promi
   })
 
   await dleet(derivedDataPath)
-  await makeDir(derivedDataPath)
```

Comment:
I removed this line, because from xcode 12.5, the build was failing with error message `*** CLEAN FAILED ***`. It seems that since xcode 12.5 the `xcodebuild` command prefers to create the folder itself.
Without it everything works fine, even after removing node_modules and rebox folders.